blender-cli/src/config.rs

```diff
@@ -91,6 +91,10 @@ pub(super) struct DepositCmd {
     /// Contract metadata file.
     #[clap(default_value = "blender-metadata.json", value_parser = parsing::parse_path)]
     pub metadata_file: PathBuf,
+
+    /// raw pk bytes file.
+    #[clap(default_value = "pk.bytes", value_parser = parsing::parse_path)]
+    pub proving_key_file: PathBuf,
```

Comment:
we need two different paths for `deposit.pk.bytes` and `withdraw.pk.bytes`

Reply:
both commands take a key argument